prime-angular-frontend/src/app/lib/modules/ngx-contextual-help/contextual-help/contextual-help.component.html

```diff
@@ -1,20 +1,27 @@
 <a mat-icon-button
-   color="primary"
+   [color]="menuIconColor"
    aria-label="Help"
+   [class.small]="small"
    [matMenuTriggerFor]="help"
    (click)="$event.stopPropagation()">
-  <mat-icon>info</mat-icon>
+  <mat-icon>{{ icon }}</mat-icon>
 </a>
 <mat-menu #help="matMenu"
-          class="p-3"
+          [class]="advanced ? 'advanced-context p-3' : 'p-3'"
           [xPosition]="xPosition"
           [yPosition]="yPosition">
   <ng-template matMenuContent>
     <!-- Content project contextual help components when available -->
-    <h1 *ngIf="contextualHelpTitleChildren.length"
-        class="title lead">
-      <ng-content select="[appContextualTitle]"></ng-content>
-    </h1>
+    <div *ngIf="contextualHelpTitleChildren.length"
+         class="d-flex align-items-stretch mb-2">
```

Comment:
Looking at the UI feels like this should be `mb-3` to create a bit more space between the title and content.